common/src/main/java/gov/cms/ab2d/common/service/WorkerDrive.java

```diff
@@ -1,4 +1,4 @@
-package gov.cms.ab2d.worker.service;
+package gov.cms.ab2d.common.service;
 
 public enum WorkerDrive {
```

Comment:
Why is this moving to common?

Reply:
So that it could be used both by HPMS and by the Worker sub-models.